src/components/QuickOpenModal.js

```diff
@@ -121,41 +122,62 @@ export class QuickOpenModal extends Component<Props, State> {
     this.setState({ results });
   };
 
+  showShortcuts = (query: string) => {
+    this.setState({ results: formatShortcutResults() });
+  };
+
   updateResults = (query: string) => {
     if (this.isSymbolSearch()) {
       return this.searchSymbols(query);
     }
+    if (this.isShortcutQuery()) {
+      return this.showShortcuts(query);
+    }
     return this.searchSources(query);
   };
 
+  setModifier = (item: QuickOpenResult) => {
+    const { setQuickOpenQuery } = this.props;
+    if (item.id === QUICKOPEN_MODIFIERS["@"]) {
```

Comment:
how about `if(["@","#",":"].includes(item.id)) setModifiers(item.id)`

Reply:
Good call if, the IDs are changed to the modifiers it works great.